arch/arm/mach-exynos/exynos.c

```diff
@@ -214,7 +214,7 @@ MACHINE_END
 		.init_early	= exynos_firmware_init,		\
 		.init_irq	= exynos_init_irq,		\
 		.init_machine	= exynos_dt_machine_init,	\
-		.init_late	= exynos_init_late,		\
+		.init_late	= exynos_pm_init,		\
```

Comment:
what's the benefit of this patch?

Reply:
After adding our environments by cherry-picking, this patch just replaces the contents.

exynos_init_late was added by another commit, having just one condition code, wrapping exynos_pm_init function. That seems to have nothing to do with our environment.

This commit: https://github.com/hardkernel/linux/commit/2edb36c4ea09fe5077e2d19e60fd353ab1fc6eee